### PR TITLE
Fix some issues in information schema docs

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

1. Fixes some issues from https://github.com/pingcap/docs/pull/3178/ and https://github.com/pingcap/docs/pull/3322;
2. Align https://github.com/pingcap/docs-cn/pull/3470 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3470 
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: information-schema/information-schema-processlist.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ summary: Learn the `PROCESSLIST` information_schema table.
 
 The `PROCESSLIST` table has additional columns not present in `SHOW PROCESSLIST`:
 
-* A `MEM` column to show memory occupied by the request being handled (in bytes)
+* A `MEM` column to show the memory used by the request that is being processed, in bytes. (in bytes)
```
Comment (yikeke) [inline information-schema/information-schema-processlist.md:12]:
```suggestion
* A `MEM` column to show the memory used by the request that is being processed (in bytes).
```

File: information-schema/information-schema-processlist.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ Fields in the `PROCESSLIST` table are described as follows:
 * STATE: The current connection state.
 * INFO: The requested statement that is being processed.
 * MEM: The memory used by the request that is being processed, in bytes.
+* TxnStart: The start time of the transaction
```
Comment (yikeke) [inline information-schema/information-schema-processlist.md:72]:
```suggestion
* TxnStart: The start time of the transaction.
```

**Comment by nullnotnil** [comment]
LGTM

**Comment by nullnotnil** [comment]
LGTM

**Comment by yikeke** [comment]
PTAL @lilin90

**Comment by nullnotnil** [comment]
LGTM, I think we can merge.

**Pull request merged**